packages/connect-web-bench/README.md

```diff
@@ -11,4 +11,4 @@ it like a web server would usually do.
 | code generator | bundle size        | minified               | compressed           |
 |----------------|-------------------:|-----------------------:|---------------------:|
 | connect-web    | 87,973 b | 41,027 b | 11,368 b |
-| grpc-web       | 394,473 b    | 283,740 b    | 51,793 b |
```

Comment:
Not sure why grpc-web's bundle size increased.

Reply:
Our `setversion` make target has the unfortunate side effect of updating dependencies, and the package `google-protobuf` was bumped from v3.21.0 to v3.21.2, increasing the bundle size. I'm filing a ticket...